velox/exec/tests/TableWriteTest.cpp

```diff
@@ -54,6 +55,29 @@ std::string testModeString(TestMode mode) {
   }
 }
 
+static std::shared_ptr<core::AggregationNode> generateAggregationNode(
+    std::string name,
+    std::vector<core::FieldAccessTypedExprPtr> groupingKeys,
+    AggregationNode::Step step,
+    PlanNodePtr source) {
+  core::TypedExprPtr inputField =
+      std::make_shared<const core::FieldAccessTypedExpr>(BIGINT(), name);
+  auto callExpr = std::make_shared<const core::CallTypedExpr>(
+      BIGINT(), std::vector<core::TypedExprPtr>{inputField}, "min");
```

Comment:
We only test min for now?

Reply:
Yes, for now we only focus on the structure test, in the followup we can start testing for different functions